.markdownlint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "MD013": false,
+  "MD041": false
```

Comment:
I can't use jsonc here, but the rules being disabled are line length and "first header must be level 1". I chose those based on violations I saw in the codebase which seemed worth overlooking.